static/js/index.js

```diff
@@ -1,3 +1,13 @@
+function github(){
```

Comment:
Let's call this something like `renderGitHubCommits()` or something like that to make it clear what this function is doing